### DISCUSSION TITLE
Grid design on frontend

### DISCUSSION THREAD

**Discussion by ars3nicum** [Budibase/budibase#10574] [General]
Will it be possible to show tables on the frontend in the same grid design system like in the new version of backend?

**Comment by melohagan** [comment]
Hey @ars3nicum 

That is the plan for future releases 😄 

Currently we're collecting feedback on the Grid view within the **Data** section before making it available in the **Design** section.

**Comment by ars3nicum** [comment]
Thanks! It would be great.
